lib/nrf_modem_lib/Kconfig

```diff
@@ -17,7 +17,11 @@ menuconfig NRF_MODEM_LIB
 	help
 	  Use Nordic Modem library.
 
-if NRF_MODEM_LIB
+config NRF_MODEM_LIB_TEST
+    bool "Enable Modem Library testing"
+    depends on BOARD_NATIVE_POSIX
+
+if NRF_MODEM_LIB || NRF_MODEM_LIB_TEST
```

Comment:
Why is it needed to add modemlib configuration to tests?

Reply:
For the numeric parameters like `CONFIG_NRF91_SOCKET_SEND_SPLIT_LARGE_BLOCKS` which are used inside the unit under test.

Reply:
I see, thanks for straightening this. Personally, I think it'd be better to define missing configs at the test level (inside CMakeLists.txt) instead of modifying existing sources (especially that current proposal limits the test usage to native_posix board only).

Reply:
Yeah, I was thinking about that. Makes more sense to be honest.